logos/src/logos/classification/classify_token.py

```
@@ -0,0 +1,14 @@
+"""
+Classifier using keywords in prompts.
+"""
+from typing import List
+from logos.classification.classifier import Classifier
+
+
+class TokenClassifier(Classifier):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)